vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -176,7 +174,12 @@ int main(int argc, char** argv)
             return l_rc;
         }
 
-        // ToDo: implementation of write keyword
+        bool l_isHardwareOperation = (l_hardwareFlag->empty() ? false : true);
```

Comment:
We can directly pass `!l_hardwareFlag->empty()` to `writeKeyword` API instead of creating this variable?

Reply:
I did not get this? Why it con;t be used in place even if "not" value is being sent.

Reply:
updated